src/fitnesse/slim/fixtureInteraction/SimpleInteraction.java

```diff
@@ -183,10 +211,31 @@ public MethodExecutionResult findAndInvoke(String methodName, Object instance, O
     Method method = findMatchingMethod(methodName, instance, args);
     if (method != null) {
       return this.invokeMethod(instance, method, args);
+    } else {
+      MethodExecutionResult mER = invokeStaticMethod(methodName, pathsCache, args );
+      if (mER != null) {
+        return mER;
+      }
     }
     return MethodExecutionResult.noMethod(methodName, instance.getClass(), args.length);
   }
 
+  private MethodExecutionResult invokeStaticMethod(String methodName, List<String> paths, Object... args) throws Throwable {
+    Method method;
+    int i = methodName.lastIndexOf('.');
+    if (i >=0) {
+      // Static Method
+      String className = methodName.substring(0, i);
+      String staticMethodName = methodName.substring(i+1);
+      Class<?> clazz = searchPathsForClass(paths, className);
+      if (clazz != null) {
+        method = findMatchingMethod(new String[]{staticMethodName}, clazz.getMethods(), args.length);
+	return this.invokeMethod(null, method, args);
```

Comment:
Why is this line not indented more?

More importantly: I believe you should check whether method is not null before trying to invoke it...

Reply:
good point, added a check